cmd/config-downloader/downloader.go

```diff
@@ -187,7 +185,8 @@ func main() {
 
 	locationArray := strings.SplitN(downloadLocation, locationSeparator, 2)
 	if locationArray == nil || len(locationArray) < 2 && downloadLocation != locationDefault {
-		panic(fmt.Sprintf("downloadLocation %s is malformated.\n", downloadLocation))
+		errorMessage := fmt.Sprintf("E! downloadLocation %s is malformated.", downloadLocation)
+		log.Panicf(errorMessage)
```

Comment:
If we're using `Panicf`, we don't need to use `fmt.Sprintf` to create the error message beforehand.

Reply:
Fixed it in the newest commit. Nice catch on these though.